test/kuttl/tests/cinder_scale/02-scale-up-cinder-api.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      oc patch cinder -n $NAMESPACE  cinder --type='json' -p=\
```

Comment:
I think you have too many references to "cinder" on this line. See the error message at test/kuttl/tests/cinder_scale